### ISSUE TITLE
Failed to "npm run bootstrap" due to no @google/model-viewer-shared-assets package

### ISSUE THREAD

**Issue by takahirox** [google/model-viewer#1021]
### Description

I tried to built &lt;model-viewer&gt locally but I face an error in "npm run bootstrap" due to no @google/model-viewer-shared-assets.

```sh
$ git clone https://github.com/google/model-viewer.git
$ cd model-viewer
$ npm install
$ npm run bootstrap

> model-viewer-packages@ bootstrap C:\Users\Takahiro\Documents\model-viewer
> lerna bootstrap --ci

lerna notice cli v3.20.2
lerna info versioning independent
lerna info ci enabled
lerna info Bootstrapping 5 packages
lerna info Installing external dependencies
lerna ERR! npm install exited 1 in '@google/model-viewer-render-fidelity-tools'
lerna ERR! npm install stderr:
npm ERR! code E404
npm ERR! 404 Not Found: @google/model-viewer-shared-assets@undefined

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Takahiro\AppData\Roaming\npm-cache\_logs\2020-02-12T20_46_09_511Z-debug.log

lerna ERR! npm install exited 1 in '@google/model-viewer-render-fidelity-tools'
lerna WARN complete Waiting for 3 child processes to exit. CTRL-C to exit immediately.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! model-viewer-packages@ bootstrap: `lerna bootstrap --ci`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the model-viewer-packages@ bootstrap script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Takahiro\AppData\Roaming\npm-cache\_logs\2020-02-12T20_46_09_556Z-debug.log
```

Actually, https://www.npmjs.com/package/@google/model-viewer-shared-assets is 404. Please let me know if I'm missing something.

Platform: Windows 10 + MINGW

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [x] Windows

**Comment by cdata** [comment]
@takahirox unfortunately we don't work on Windows, so it's difficult for us to know for sure if there are Windows-specific quirks with our dev process. The package you are referring to is a "private" package (found in `packages/shared-assets`), so we don't publish it to NPM. Instead, our package management tool [Lerna is supposed to create a symlink when you run bootstrap](https://lerna.js.org/#command-bootstrap).

You can see in our CI build that `npm run bootstrap` works as intended in a fresh Linux environment: https://github.com/google/model-viewer/runs/442731146?check_suite_focus=true. It also works in my local environment.

I would like to help, but I don't have access to a Windows machine right now.

**Comment by takahirox** [comment]
Thanks for the response. Regarding no @google/model-viewer-shared-assets package problem I found an easy mistaken in my side, node and npm are old. By upgrading to them, the problem has been resolved.

But I faced another problem. A script includes `ln` command, but MINGW handles it as just copy so that `if [ -L file ]` in shell script causes a problem. I also tried PowerShell but it doesn't have `ln` command.

Finally I tried WSL (Windows Subsystem for Linux) Ubuntu then it works fine, thanks!
